src/graphql.tsx

```diff
@@ -50,11 +50,11 @@ export declare interface QueryOptions {
   skip?: boolean;
 }
 
-export interface GraphQLDataProps {
+export interface QueryProps {
   error?: ApolloError;
   networkStatus: number;
   loading: boolean;
-  variables: {
+  variables?: {
```

Comment:
i think this should not have the `?` it looks like if there's no variables it's an empty object, not `undefined`